packages/automerge-repo/src/DocHandle.ts

```diff
@@ -426,6 +434,11 @@ export class DocHandle<T> extends EventEmitter<DocHandleEvents<T>> {
     this.#machine.send({ type: DELETE })
   }
 
+  /** Called by the repo to free memory used by the document. */
+  reset() {
```

Comment:
I think that rather than calling this `reset` we should all it `idle` as I think that more clearly describes what it actally does.